app/src/main/java/com/tvtracker/MainActivity.kt

```diff
@@ -437,7 +451,7 @@ class MainActivity : ComponentActivity() {
             },
             singleLine = true,
 
-            label = {Text(text= "Search for a Movie or TV Show", color= Color.White)},
+            label = {Text(text= stringResource(R.string.InsertedSearchMovieText), color= Color.White)},
```

Comment:
Converted to a string resource, loose strings are unsafe in the code labels and descriptions, and not converting is against best practices. 
https://developer.android.com/guide/topics/resources/string-resource#String